### PR TITLE
More about using static assets

### PR DESCRIPTION
**Pull request by rwaldron**
Signed-off-by: Rick Waldron <waldron.rick@gmail.com>

### CODE REVIEW

**Comment by rwaldron** [comment]
cc @HipsterBrown @Frijol

File: cli.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ If `--name` is not specified, CLI will look for an environment variable, e.g. `e
 Along with the package.json and index.js included in the `t2 init` process, there are some other files that may be useful for your project:
 * `.tesselignore` similar to .gitignore or [.npmignore](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package), this file should list any files or directories you want ignored by the T2 bundling and deployment process. This is handy when using the `--full` flag, which tells T2 to bundle everything in the project directory.
 * `.tesselinclude` the overriding and opposite behavior of `.tesselignore`, this file should list any files or directories you want included with the T2 bundling and deployment process. This is handy for including non-JavaScript assets, like HTML, CSS, and images, for use within your project.
+  * In your JavaScript program, assets listed in the `.tesselinclude` file should always be accessed using `__dirname`. For example, if there is a file at `public/index.html` and it's listed in your `.tesselinclude`, from your JavaScript program, you'd read it like this: `fs.readFile(path.join(__dirname, "public/index.html"), "utf8", (error, contents) => ...);`.
```
Comment (HipsterBrown) [inline cli.md]: Can we add some info about `__dirname`, i.e. what is it? Either linking to [the docs](https://nodejs.org/api/globals.html#globals_dirname) and/or a comment about `__dirname` being the directory of the module being executed might be helpful for explaining why it's useful referencing assets on Tessel.

Comment (rwaldron) [inline cli.md]: Can you make a PR to my branch and I will merge it in?

**Comment by HipsterBrown** [comment]
LGTM

**Pull request merged**